packages/transaction-builder/src/models/coin-denom.ts

```diff
@@ -2,6 +2,6 @@
  * An enum to represent all the valid Coin Denominations in Pocket Network
  */
 export enum CoinDenom {
-    Upokt = 'upokt',
-    Pokt = 'pokt'
+    Upokt = 'Upokt',
+    Pokt = 'Pokt'
```

Comment:
@crisog this is the biggest quirk ever haha: the autogenerated tx-signer NEEDS this to be uppercase as it's calling the `toLowerCase` function on it.